README.md

```diff
@@ -1,75 +1,72 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
+# Awesome Book milestone 1
 
-# Project Name
+> This is a simple ES6 Javascript project that builds a basic website that allows users to add/remove books from a list. This is achieved by using JavaScript objects and arrays and dynamically DOM modification and use of add basic events.
 
-> Description the project.
+## Built with
 
+- HTML CSS & JAVASCRIPT ES6
 
-## Built With
+## Additional Tools
 
-- Major languages
-- Frameworks
-- Technologies used
+- WebHint
+- Stylelint
+- ESlint
 
-## Live Demo (if available)
+## Live demo
 
-[Live Demo Link](https://livedemo.com)
+You can watch [here](/) a live demo of our project.
 
+## Test
 
-## Getting Started
-
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-
-To get a local copy up and running follow these simple example steps.
+To track linter errors locally follow these steps:
 
-### Prerequisites
+Download all the dependencies run:
 
-### Setup
+```
+npm install
+```
 
-### Install
+Track HTML linter errors run:
 
-### Usage
+```
+npx hint .
+```
 
-### Run tests
+Track CSS linter errors run:
 
-### Deployment
+```
+npx stylelint "**/*.{css,scss}"
+```
 
+Track JavaScript linter errors run:
 
+```
+npx eslint ."
+```
 
 ## Authors
 
-👤 **Author1**
+👤 **Petro Loltolo Lesapiti**
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+- GitHub: [@petrolesapiti](https://github.com/Loltolo-Lesapiti)
+- LinkedIn: [@petrolesapiti](https://www.linkedin.com/in/petrolesapitiloltolo/)
 
-👤 **Author2**
-
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+## Collaborators
 
 ## 🤝 Contributing
 
 Contributions, issues, and feature requests are welcome!
 
-Feel free to check the [issues page](../../issues/).
-
-## Show your support
+## Getting Started
 
-Give a ⭐️ if you like this project!
+To get a local copy follow these simple steps:
```

Comment:
- [OPTIONAL ] Please it will be nice to add explicit steps to clone and run your project locally, as this will be of help to non-technical users. You can put the steps to clone and run the project in a new Getting Started section under the project's description. This includes `git clone URL`,  `npm start` etc 😉